### ISSUE TITLE
hard-coded database name in v0.17.0

### ISSUE THREAD

**Issue by vranystepan** [google/exposure-notifications-server#1194]
### TL;DR

During the upgrade process, we discovered quite a big drift caused by the changed database name
https://github.com/google/exposure-notifications-server/blob/v0.17.0/terraform/database.tf#L105

**Expected behavior**

in previous versions, we were able to supply database name in variables

**Observed behavior**

DB name is hard-coded to `key` so it's causing replacement of database. 

### Reproduction

`terraform apply / plan` with https://github.com/google/exposure-notifications-server/releases/tag/v0.17.0

**Additional information**
can we return variable `db_name`, please?

**Comment by vranystepan** [comment]
same for db user https://github.com/google/exposure-notifications-server/commit/75d0423e5dbee413f75e60737cc9cca9329fd5c2#diff-e74570c17a49307ad3a889df543757969e5b0234b8e255bce28c2380033eb24eL89 

I might submit PR once I'm done with the upgrade of our infrastructure :)

**Comment by sethvargo** [comment]
Note there's a difference between the database name and the database _instance_ name. We needed to remove the database _instance_ name to allow Terraform to properly manage multiple Cloud SQL instances and to support read-only replicas.

The name of the actual database itself (in postgres) was never customizable via a variable. We needed to change it to accommodate some upcoming changes.

We'd accept a PR to parameterize db_name and db_user, but not one to parameterize db_instance_name. Thanks!
